src/vs/workbench/contrib/notebook/browser/contrib/editorStatusBar/editorStatusBar.ts

```diff
@@ -283,6 +283,10 @@ export class NotebookIndentationStatus extends Disposable implements IWorkbenchC
 		const activeEditor = getNotebookEditorFromEditorPane(this._editorService.activeEditorPane);
 		if (activeEditor) {
 			this._show(activeEditor);
+			this._register(activeEditor.onDidChangeActiveCell(() => {
```

Comment:
It seems we creating a disposable for this event every time when we change editor focuses, editor/notebook settings change. We may want to add it to `_itemDisposables`.